hyprctl/main.cpp

```diff
@@ -351,6 +352,8 @@ int main(int argc, char** argv) {
             } else if ((ARGS[i] == "-f" || ARGS[i] == "--follow") && !fullArgs.contains("f")) {
                 fullArgs += "f";
                 needRoll = true;
+                if (ARGS.size() >= 3)
+                    rollDelay = std::stoi(ARGS[2]);
```

Comment:
it is not such comfortable to simple users, to count milliseconds

maybe max rps (requests per second) option will be more easy to understand, like fps in games

Reply:
sure, done